src/App.js

```diff
@@ -1,26 +1,32 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
 
-export class App extends Component {
-  state = {
-    movies: data,
+export const App = () => {
+  const [movies, setMovie] = useState(data);
+  const [addErrorMessage, getAddError] = useState(true);
+  const addMovie = (movie) => {
+    if (!movies.find(item => item.imdbId === movie.imdbId)) {
+      setMovie([...movies, movie]);
```

Comment:
```suggestion
    if (!movies.some(item => item.imdbId === movie.imdbId)) {
      setMovie([...movies, movie]);
```
метод some тут підійшов би краще

Reply:
а почему?

Reply:
А зачем тебе find, если ты с найденным объектом ничего не будешь делать потом? Тебе надо просто знать есть такой объект или нет, и метод some как раз возвращает true/false